SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -58,8 +60,20 @@ public extension I18nManager {
 
 public extension I18nManager {
     
-    func localizedString(forKey key: String) -> String {
-        return localizationPerformingBlock(key, language)
+    func localizedString(forKey key: String, language: String? = nil) -> String {
+
+        var localizedString = localizationPerformingBlock(key, language ?? self.language)
```

Comment:
```suggestion
    func localizedString(forKey key: String, language: String) -> String {

        var localizedString = localizationPerformingBlock(key, language)
```

Why did you change this? It will introduce breaking change that we want to avoid.

Reply:
Can you explain, I don't see a breaking change? Before there was no language parameter, and it worked with language property which is a member of I18nManager, I don't see a major change here with this optional parameter. 
But if it is a big concern I can implement a different method for this

Reply:
We reviewed the comments and decided to go wi the changes and add some comments to method declaration